### PR TITLE
Adding BIOS Parameters in api.md

### PR DESCRIPTION
**Pull request by Hellcatlk**
Signed-off-by: Zou Yu <zouy.fnst@cn.fujitsu.com>

### CODE REVIEW

**Comment by Hellcatlk** [comment]
@andfasano @zaneb

**Comment by andfasano** [comment]
Thanks @Hellcatlk , /lgtm

File: docs/api.md [hunk 1]
```diff
@@ -182,6 +182,24 @@ The sub-fields are:
     GiB. If unspecified or set to 0, the maximum capacity of disk will be
     used for logical disk.
 
+#### bios
```
Comment (dtantsur) [inline docs/api.md:185]: It's called `firmware`, not `bios`

Comment (Hellcatlk) [reply docs/api.md]: Thanks, done.

**Pull request merged**